### DISCUSSION TITLE
Duplicate "Prev Page" and "Next Page" as arrows below model cards

### DISCUSSION THREAD

**Discussion by Terroosh** [BlafKing/sd-civitai-browser-plus#93] [Ideas]
![arrow](https://github.com/BlafKing/sd-civitai-browser-plus/assets/55752005/3deef062-b803-4a77-857b-8f56c126b8ef)

Scrolling back to the top of the page to click "Next Page" can be a bit tedious when tile count is set high and especially when a model has a long description. It would be neat if you could duplicate the "Prev Page" and "Next Page" buttons but as arrows in a small box like the magnifier icon and then set them similar to the picture above.

**Comment by BlafKing** [comment]
I'd like to avoid adding more buttons to avoid the UI becoming cluttered like it was before.
I would however like to find an alternative that could work for you :)

In the next update, I'll add an extra setting to the CivitAI Browser settings tab that will allow you to keep the page controls sticked to the top as a header like so:
![image](https://github.com/BlafKing/sd-civitai-browser-plus/assets/9644716/920e4be7-983d-49c3-9ec3-91eb0aafca21)

Additionally, I will make sure that a button pops up in the bottom right if you have scrolled down that can be used to go back to the top instantly by clicking it! 👍🏻

**Comment by Terroosh** [comment]
That's even better, thanks!

**Comment by BlafKing** [comment]
No problem! Latest version (v1.15) now includes this functionality!

I'll close this thread since your request has been completed! :)
